packages/firebase_remote_config/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -0,0 +1,118 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
@kroikie I think a lot of the public methods aren't flagging that code comments are missing because of the analyzer.yaml file.

Reply:
Good catch, I added the docs to the platform interface level definitions and I assumed that would be sufficient. I will add them here as well.

I wonder if there is a way to avoid duplicating the definitions across multiple files, they are the same from what I can tell.

Reply:
Unfortunately not since the class isn't extended. I just copy paste them across

Reply:
Done!